tests/other/test_smtlibv2.py

```diff
@@ -446,20 +742,29 @@ def testBasicArrayProxySymbIdx2(self):
 
         solutions = self.solver.get_all_values(cs, array[0])  # get a concrete solution for index
         self.assertItemsEqual(solutions, (1, 2))
-
         solutions = self.solver.get_all_values(
-            cs, array.get(0, 100)
+            cs, array.select(0)
         )  # get a concrete solution for index 0
         self.assertItemsEqual(solutions, (1, 2))
 
         solutions = self.solver.get_all_values(
-            cs, array.get(1, 100)
+            cs, array.select(1)
         )  # get a concrete solution for index 1 (default 100)
         self.assertItemsEqual(solutions, (100, 2))
 
-        self.assertTrue(
-            self.solver.can_be_true(cs, array[1] == 12345)
-        )  # no default so it can be anything
+    def testBasicConstatArray(self):
+        cs = ConstraintSet()
+        array1 = MutableArray(
+            cs.new_array(index_size=32, value_size=32, length=10, name="array1", default=0)
+        )
+        array2 = MutableArray(
+            cs.new_array(index_size=32, value_size=32, length=10, name="array2", default=0)
+        )
+        array1[0:10] = range(10)
+        self.assertTrue(array1[0] == 0)
+        # yeah right self.assertTrue(array1[0:10] == range(10))
+        array_slice = array1[0:10]
+        self.assertTrue(array_slice[0] == 0)
```

Comment:
We also want to add another test to this, right?
```suggestion
        self.assertEqual(array_slice[0], 0)
        self.assertEqual(array_slice[9], 9)
```